config/config.yaml

```diff
@@ -254,27 +254,7 @@ tide:
     - jetstack/cert-manager
     excludedBranches:
     - master
```

Comment:
We don't want to exclude any branches from this config else tide won't work on those branches. We only use `include/exclude` when code freeze is enabled in order to have different config per branch

Reply:
I see!